v2/event/content_type.go

```diff
@@ -14,6 +14,20 @@ const (
 	ApplicationCloudEventsBatchJSON = "application/cloudevents-batch+json"
 )
 
+type ContentType string
```

Comment:
Hm, can we 1/ not export this and 2/ simplify this into just a simple function? See below.